app/src/test/java/org/simple/clinic/medicalhistory/MedicalHistoryTest.kt

```diff
@@ -0,0 +1,55 @@
+package org.simple.clinic.medicalhistory
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.TestData
+import org.simple.clinic.medicalhistory.Answer.Unanswered
+import org.simple.clinic.medicalhistory.Answer.Yes
+import java.util.UUID
+
+class MedicalHistoryTest {
+
+  @Test
+  fun `if diabetes and hypertension is not answered then diagnosis required is false`() {
+    val medicalHistory = TestData.medicalHistory(
+        uuid = UUID.fromString("0bd13198-2dca-4a06-b795-6ee45ef8572b"),
+        diagnosedWithHypertension = Unanswered,
+        hasDiabetes = Unanswered
+    )
+
+    assertThat(medicalHistory.diagnosisRecorded).isFalse()
+  }
+
+  @Test
+  fun `if diabetes is answered and hypertension is not answered then diagnosis required is false`() {
+    val medicalHistory = TestData.medicalHistory(
+        uuid = UUID.fromString("0bd13198-2dca-4a06-b795-6ee45ef8572b"),
+        diagnosedWithHypertension = Unanswered,
+        hasDiabetes = Yes
+    )
+
+    assertThat(medicalHistory.diagnosisRecorded).isFalse()
+  }
+
+  @Test
+  fun `if diabetes is not answered and hypertension is answered then diagnosis required is false`() {
+    val medicalHistory = TestData.medicalHistory(
+        uuid = UUID.fromString("0bd13198-2dca-4a06-b795-6ee45ef8572b"),
+        diagnosedWithHypertension = Yes,
+        hasDiabetes = Unanswered
+    )
+
+    assertThat(medicalHistory.diagnosisRecorded).isFalse()
+  }
+
+  @Test
+  fun `if diabetes and hypertension is answered then diagnosis required is true`() {
+    val medicalHistory = TestData.medicalHistory(
+        uuid = UUID.fromString("0bd13198-2dca-4a06-b795-6ee45ef8572b"),
+        diagnosedWithHypertension = Yes,
+        hasDiabetes = Yes
+    )
+
+    assertThat(medicalHistory.diagnosisRecorded).isTrue()
+  }
```

Comment:
Nice! Can I ask you do a couple of small changes to these tests, please?

- Change all the test method names to "should be" instead of "is" when declaring should happen. Ex: `if diabetes and hypertension is answered then diagnosis required should be true`